castervoice/apps/dragon.py

```diff
@@ -55,24 +89,76 @@ class DragonRule(MergeRule):
         "center point":
             R(Playback([(["MouseGrid"], 0.1), (["click"], 0.0)]),
               rdescript="Mouse: Center Point"),
-        # A Natlink Command
-        "clear caster log": R(Function(utilities.clear_log), rdescript="Core: Clear Caster Log"),
+        "[show] windows": Mimic("list", "all", "windows"), # this emulates a useful native dragon command
+        "cory <dict>": 
+            R(Mimic("correct", extra="dict") + WaitWindow(title="spelling window") + Mimic("choose", "one"),
+                rdescript="brings up the correction menu for the phrase spoken in the command and chooses the 1st choice"),
+        "cory that": 
+            R(Mimic("correct", "that") + WaitWindow(title="spelling window") + Mimic("choose", "one"), 
+                rdescript="brings up the correction menu for the previously spoken phrase"),
+
+        "make that <dict>": R(Mimic("scratch", "that") + Mimic(extra="dict"), 
+             rdescript="deletes the dictation generated by the previous utterance and replaces it with what you say next"),
+        'strike [<n>]': R(Playback([(["scratch", "that"], 0.03)]), 
+            rdescript="delete dictation from previous n utterances") * Repeat(extra="n"),
+             
+        "train word": R(Mimic("train", "that") + Key("a-r/200, s"),
+             rdescript="quickly train word when you have it selected in a Dragon friendly text field"),
```

Comment:
Any way to get this to work in a non-Dragon friendly context? Perhaps using `Mimic`?